### PR TITLE
GG-36062 CacheDataRegionConfigurationTest#doTestLargeRegionsWithEviction crashes agents on TС due to large memory allocation

### PR DESCRIPTION
**Pull request by rpuch**
https://ggsystems.atlassian.net/browse/GG-36062

### CODE REVIEW

File: modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/evict/RandomLruPageEvictionTrackerTest.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+package org.apache.ignite.internal.processors.cache.persistence.evict;
+
+import org.junit.Test;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+/**
+ */
```
Comment (tkalkirill) [inline modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/evict/RandomLruPageEvictionTrackerTest.java:25]: Can we add a description?

File: modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/evict/Random2LruPageEvictionTrackerTest.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+package org.apache.ignite.internal.processors.cache.persistence.evict;
+
+import org.junit.Test;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+/**
+ */
```
Comment (tkalkirill) [inline modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/evict/Random2LruPageEvictionTrackerTest.java:25]: Can we add a description?

File: modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/evict/Random2LruPageEvictionTrackerTest.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+public class Random2LruPageEvictionTrackerTest {
+    @Test
+    public void trackingArraySizeMayExceed2Gb() {
+        assertThat(Random2LruPageEvictionTracker.trackingArraySize(1024 * 1024 * 1024), is(8L * 1024L * 1024L * 1024L));
```
Comment (tkalkirill) [inline modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/evict/Random2LruPageEvictionTrackerTest.java:29]: Maybe also test `trackingArrayOffset` ?

File: modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/evict/RandomLruPageEvictionTrackerTest.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+public class RandomLruPageEvictionTrackerTest {
+    @Test
+    public void trackingArraySizeMayExceed2Gb() {
+        assertThat(RandomLruPageEvictionTracker.trackingArraySize(1024 * 1024 * 1024), is(4L * 1024L * 1024L * 1024L));
```
Comment (tkalkirill) [inline modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/evict/RandomLruPageEvictionTrackerTest.java:29]: Maybe also test `trackingArrayOffset` ?

**Review by SammyVimes — APPROVED** [review]
LGTM

**Pull request merged**